docs/integrations/airbyte.md

```diff
@@ -30,7 +29,7 @@ The integration between the two open-source projects brings resilience and manag
 connectors to sync data to your S3 buckets by leveraging lakeFS branches and atomic commits and merges.
 
 ## Use cases
-You can take advantage of lakeFS consistency guarantees and [CI/CD capabilities](../using_lakefs/ci.md) when ingesting data to S3 using lakeFS:
+You can take advantage of lakeFS consistency guarantees and [CI/CD capabilities](../data_lifecycle_management) when ingesting data to S3 using lakeFS:
```

Comment:
Update the title of the link 'CI/CD...' to match the page, "Data Lifecycle Management"